modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,172 @@
+
+```
+curl -X POST http://localhost:9644/v1/topics/mount -d {
+  "topics": [
+    {
+      "source_topic": {"ns": "kafka", "topic": "<source-topic-1-name>"}, 
+      "alias": {"ns": "kafka", "topic": "<new-topic-1-name>"}
+    }, 
+    {
+      "source_topic": {"ns": "kafka", "topic": "<source-topic-2-name>"}
+    }, 
+    {
+      "source_topic": {"ns": "kafka", "topic": "source-topic-3-name"}, 
+      "alias": {"ns": "kafka", "topic": "<new-topic-3-name>"}
+    }
+  ]
+}
+```
+
+* `ns` is the topic namespace. This field is optional. Default value: `kafka`
+* To rename a topic in the target cluster, use the optional `alias` object in the request body. In the example, topics 1 and 3 are given new names in the target cluster, while topic 2 retains its original name.
+
+--
+
+======
+
+When the migration is complete, the target cluster handles produce and consume workloads for the topics.
+
+== Monitor progress
+
+Issue a GET request to the `/migrations` endpoint to view the state of topic migrations:
+
+```
+curl http://localhost:9644/v1/migrations 
+```
+
+The response returns the state of existing migrations:
+
+|===
+| Migration state | Outbound migration (unmount) | Inbound migration (mount)
+
+| `planned`
+| Redpanda has initialized a migration.
+|
+
+| `preparing`
+| Redpanda copies all possible topic data including topic manifests to the destination bucket or container in object storage.
+| Redpanda recreates the topics in a disabled state in the target cluster. The cluster allocates partitions but does not add log segments yet. Topic metadata is populated from the topic manifests found in object storage.
+
+| `prepared` 
+| The migration is ready to execute. In this state, the cluster still accepts client reads and writes for the topics.
+| 
+
+| `executing` 
+| The cluster rejects client reads and writes for the topics. Redpanda uploads any remaining topic data that has not yet been copied to object storage.
+| The target cluster starts topic partitions with log segments downloaded from object storage.
+
+| `executed` 
+| All unmounted topic data from the cluster is available in object storage.
+| All mounted topic data is available in the target cluster.
+
+| `cut_over`
+| The migration is complete and can be deleted. Redpanda deletes the unmounted topics from the cluster.
+| The migration is complete and can be deleted.
+
+| `finished`
+| The migration is deleted. 
+| The migration is deleted. The target cluster starts to handle produce and consume workloads.
+
+| `canceling`
+| Redpanda is in the process of canceling the migration.
+|
+
+| `cancelled`
+| The migration is canceled.
+|
+
+|===
+
+== Cancel a migration
+
+You can cancel a topic migration by running this command:
+
+```
+curl -X POST http://localhost:9644/v1/<migration-id>/?action=cancel
+```
+
+You can only cancel a migration that is not in the following final <<monitor-progress,states>>:
```

Comment:
You also cannot cancel a migration in `planned` state (but you can delete it). Neither can you cancel a migration in `cancelled` or `canceling` state.